client/flutter/lib/api/who_service.dart

```diff
@@ -57,11 +58,16 @@ class WhoService {
     var headers = {
       'Content-Type': 'application/json',
       'Who-Client-ID': clientId,
-      'Who-Platform': _platform
+      'Who-Platform': _platform,
+      'User-Agent': userAgent,
     };
     return headers;
   }
 
+  static String get userAgent {
+    return 'WHO-App/${_platform}/${packageInfo != null ? packageInfo.version : ''}/${packageInfo != null ? packageInfo.buildNumber : ''} (gzip)';
```

Comment:
When would `packageInfo` be equal to `null`?

Reply:
On web https://github.com/WorldHealthOrganization/app/blob/master/client/flutter/lib/main.dart#L27